packages/web/src/components/user-card/UserCard.tsx

```diff
@@ -45,8 +52,14 @@ export const UserCard = (props: UserCardProps) => {
 
   return (
     <Card size={size} onClick={handleClick} {...other}>
-      <Avatar userId={id} aria-hidden p='l' pb='s' />
-      <Flex direction='column' p='s' pt={0} gap='xs'>
+      <Avatar
+        userId={id}
+        aria-hidden
+        p='l'
+        pb='s'
+        imageSize={avatarSizeMap[size]}
+      />
```

Comment:
Why doesn't this belong inside CardContent?

Reply:
yeah it's a bit awkward... the images have very different padding and resize considerations. I could have a CardImage wrapper to make that more explicit?

Reply:
or maybe change the name from CardContent to CardText or something like that to make it more clear?